control_plane/src/endpoint.rs

```diff
@@ -137,6 +145,36 @@ impl ComputeControlPlane {
             .unwrap_or(self.base_port)
     }
 
+    /// Create a JSON Web Key Set. This ideally matches the way we create a JWKS
+    /// from the production control plane.
+    fn create_jwks_from_pem(pem: Pem) -> Result<JwkSet> {
```

Comment:
Do we need a Pem? Maybe, reading the key as bytes is an option so that we wouldn't have extra code?